### PR TITLE
added 'q' to quit prompt

### PR DESCRIPTION
**Pull request by zeachco**
maybe having "q" to exit feel more intuitive being in git prompt?

### CODE REVIEW

File: bin/git-bc [hunk 1]
```diff
@@ -20,10 +20,10 @@ if (opts.indexOf('-h') > -1) {
   gitBc.on('success', log.success);
 
   // Exits program execution on ESC keypress
-  process.stdin.on('keypress', function (ch, key) {
-    if (key && key.name === 'escape') {
+  process.stdin.on('keypress', function(ch, key) {
```
Comment (ruyadorno) [inline bin/git-bc]: oh, this line got changed for no reason at all :disappointed: plus it breaks my standard of always having a space between `function` declaration and parenthesis :)

other than that LGTM :+1:

Comment (zeachco) [inline bin/git-bc]: Yeah, my bad for the space, also I was looking for autocomplete and thought if was more readable if we added multiples functions like autocomplete searches after to just separate on 2 lines the key verification but as this commit is only to add the Q key, i removed them :+1:

File: bin/git-bc [hunk 2]
```diff
@@ -21,9 +21,8 @@ if (opts.indexOf('-h') > -1) {
 
   // Exits program execution on ESC keypress
   process.stdin.on('keypress', function (ch, key) {
-    if (key && key.name === 'escape') {
+    if (key && (key.name === 'escape' || key.name === 'q')) {
       process.exit();
     }
   });
 }
-
```
Comment (ruyadorno) [inline bin/git-bc:29]: now my EOL is gone! :fearful: LOL

Comment (ruyadorno) [inline bin/git-bc:29]: http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline

Comment (zeachco) [inline bin/git-bc:29]: EOL back

**Comment by ruyadorno** [comment]
:confetti_ball: many thanks @zeachco :smiley:

**Pull request merged**